core/src/components/input/test/basic/input.e2e.ts

```diff
@@ -130,4 +130,72 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, screenshot, c
       await expect(item).toHaveScreenshot(screenshot(`input-with-clear-button-item-color`));
     });
   });
+
+  test.describe(title('input: click'), () => {
+    test('should trigger onclick only once when clicking the label', async ({ page }) => {
+      // Create a spy function in page context
+      await page.setContent(
+        `
+        <ion-input
+          label="Click Me"
+          value="Test Value"
+        ></ion-input>
+      `,
+        config
+      );
+
+      // Track calls to the exposed function
+      const clickEvent = await page.spyOnEvent('click');
+      const input = page.locator('label.input-wrapper');
+
+      // Use position to make sure we click into the label enough to trigger
+      // what would be the double click
+      await input.click({
+        position: {
+          x: 5,
+          y: 5,
+        },
+      });
+
+      // Verify the click was triggered exactly once
+      expect(clickEvent).toHaveReceivedEventTimes(1);
+
+      // Verify that the event target is the checkbox and not the item
+      const event = clickEvent.events[0];
+      expect((event.target as HTMLElement).tagName.toLowerCase()).toBe('ion-input');
+    });
+
+    test('should trigger onclick only once when clicking the wrapper', async ({ page }) => {
```

Comment:
This test passes when I remove the changes to `input.tsx` and I'm also unable to reproduce the double click with the wrapper.

Reply:
Just pushed a fix for that, I accidentally removed the `label-placement` property, which kinda helps cause the issue. Good catch.